packages/rspack-test-tools/tests/hookCases/compilation#processAssets/update-asset/test.js

```diff
@@ -3,10 +3,14 @@ const { createFsFromVolume, Volume } = require("memfs");
 const outputFileSystem = createFsFromVolume(new Volume());
 let contentHashes = [];
 
+/** @type {import("../../../..").THookCaseConfig} */
 module.exports = {
 	description: "should emit assets correctly",
 	options(context) {
 		return {
+			output: {
+				filename: '[name].[contenthash].js'
```

Comment:
Test failed without `[contenthash]` because the contenthash array of main chunk is empty.